dashboard/src/app/billing/card-info/card-info.html

```diff
@@ -51,6 +52,7 @@
                che-label-name="State"
                che-name="state"
                che-place-holder=""
+               che-type-city
```

Comment:
ok but my issue is that the name of validator contains "city" while it is more generic